src/schemas/transaction.json

```diff
@@ -88,7 +88,7 @@
 						"title": "value",
 						"$ref": "#/components/schemas/uint"
 					},
-					"input": {
+					"data": {
```

Comment:
Line 69 above (`required` fields) also needs to change.

Reply:
Fixed, thanks!